src/main/java/org/fundacionjala/virtualassistant/repository/mongo/RecordingRepositoryImpl.java

```diff
@@ -0,0 +1,68 @@
+package org.fundacionjala.virtualassistant.repository.mongo;
+
+import org.bson.Document;
+import org.fundacionjala.virtualassistant.models.mongo.Recording;
+import org.fundacionjala.virtualassistant.repository.mongo.RecordingRepo;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.mongodb.core.MongoTemplate;
+import org.springframework.data.mongodb.core.query.Criteria;
+import org.springframework.data.mongodb.core.query.Query;
+import org.springframework.stereotype.Repository;
+import org.springframework.web.multipart.MultipartFile;
+
+import java.util.Base64;
+import java.util.List;
+
+@Repository
+public class RecordingRepositoryImpl implements RecordingRepo {
+
+    @Autowired
+    MongoTemplate mongoTemplate;
+
+    @Override
+    public Recording getRecording(String idRecording) {
+        Query query = new Query(Criteria.where("idRecording").is(idRecording));
```

Comment:
You are repiting this same portion of code, in almost all of the methods, so please abstract this Query creating in a separated method